docs/source-pytorch/fabric/fabric.rst

```diff
@@ -3,125 +3,100 @@ Fabric
-
-- I want to quickly scale my existing code to multiple devices with minimal code changes.
-- I would like to convert my existing code to the Lightning API, but a full path to Lightning transition might be too complex. I am looking for a stepping stone to ensure reproducibility during the transition.
-
-
-.. warning:: :class:`~lightning_fabric.fabric.Fabric` is currently a beta feature. Its API is subject to change based on your feedback.
-
-
-----------
-
-****************
-Learn by example
-****************
-
-
-My Existing PyTorch Code
-========================
-
-The ``train`` function contains a standard training loop used to train ``MyModel`` on ``MyDataset`` for ``num_epochs`` epochs.
-
-.. code-block:: python
+      import torch
+      import torch.nn as nn
+      from torch.utils.data import DataLoader, Dataset
 
-    import torch
-    from torch import nn
-    from torch.utils.data import DataLoader, Dataset
+    + from lightning.fabric import Fabric
 
+      class MyModel(nn.Module):
+          ...
 
-    class MyModel(nn.Module):
-        ...
+      class MyDataset(Dataset):
+          ...
 
+    + fabric = Fabric(accelerator="cuda", devices=8, strategy="ddp")
+    + fabric.launch()
 
-    class MyDataset(Dataset):
-        ...
+    - device = "cuda" if torch.cuda.is_available() else "cpu
+      model = MyModel(...)
+      optimizer = torch.optim.SGD(model.parameters())
+    + model, optimizer = fabric.setup(model, optimizer)
+      dataloader = DataLoader(MyDataset(...), ...)
+    + dataloader = fabric.setup_dataloaders(dataloader)
+      model.train()
 
+      for epoch in range(num_epochs):
+          for batch in dataloader:
+    -         batch.to(device)
+              optimizer.zero_grad()
+              loss = model(batch)
+    -         loss.backward()
+    +         fabric.backward(loss)
+              optimizer.step()
 
-    def train(args):
-        device = "cuda" if torch.cuda.is_available() else "cpu"
 
-        model = MyModel(...).to(device)
-        optimizer = torch.optim.SGD(model.parameters(), ...)
+.. note:: :class:`~lightning_fabric.fabric.Fabric` is currently a beta feature. Its API is subject to change based on your feedback.
 
-        dataloader = DataLoader(MyDataset(...), ...)
 
-        model.train()
-        for epoch in range(args.num_epochs):
-            for batch in dataloader:
-                batch = batch.to(device)
-                optimizer.zero_grad()
-                loss = model(batch)
-                loss.backward()
-                optimizer.step()
+With only a few changes to your code, Fabric enables for you:
 
+- Automatic placement of models and data onto the device
+- Automatic support for mixed precision (smaller memory footprint)
+- Seamless switching between hardware (CPU, GPU, TPU) and distributed training strategies
+  (data-parallel, model-parallel, etc.)
+- Automated spawning of processes, no launch utilities required
```

Comment:
maybe state that this is still compatible with launching utilities like torchelastic? This reads as they wouldn't be supported...